covasim/parameters.py

```diff
@@ -111,6 +112,10 @@ def set_contacts(pars):
         if pars.get('contacts',   None) is None: pars['contacts']   = {'a': 20}  # Number of contacts per person per day -- 'a' for 'all'
         if pars.get('beta_layer', None) is None: pars['beta_layer'] = {'a': 1.0} # Per-population beta weights; relative
         if pars.get('quar_eff',   None) is None: pars['quar_eff']   = {'a': 0.3} # Multiply beta by this factor for people who know they've been in contact with a positive, even if they haven't been diagnosed yet
+        # Set the household size
+        household_size = cvdata.loaders.get_country_household_size_average(pars['location'])
```

Comment:
This seems like the most reasonable place to put this. I am setting the household size and the poptype. I assume it needs to be in use layers mode because this is where the defualt h is being set...